visualization/app/codeCharta/ui/toolBar/globalConfigurationButton/globalConfigurationDialog/mapLayoutSelection/mapLayoutSelection.component.spec.ts

```diff
@@ -1,50 +1,53 @@
-import { TestBed } from "@angular/core/testing"
-import { fireEvent, render, screen, waitForElementToBeRemoved } from "@testing-library/angular"
-import userEvent from "@testing-library/user-event"
+import { ComponentFixture, TestBed } from "@angular/core/testing"
+import { screen } from "@testing-library/angular"
 import { LayoutAlgorithm } from "../../../../../codeCharta.model"
 import { setLayoutAlgorithm } from "../../../../../state/store/appSettings/layoutAlgorithm/layoutAlgorithm.actions"
 import { Store } from "../../../../../state/store/store"
 import { MapLayoutSelectionComponent } from "./mapLayoutSelection.component"
 import { MapLayoutSelectionModule } from "./mapLayoutSelection.module"
+import { HarnessLoader } from "@angular/cdk/testing"
+import { TestbedHarnessEnvironment } from "@angular/cdk/testing/testbed"
+import { MatSelectHarness } from "@angular/material/select/testing"
+import { MatInputHarness } from "@angular/material/input/testing"
+
+let loader: HarnessLoader
+let fixture: ComponentFixture<MapLayoutSelectionComponent>
 
 describe("MapLayoutSelectionComponent", () => {
 	beforeEach(() => {
 		Store["initialize"]()
 		TestBed.configureTestingModule({
-			imports: [MapLayoutSelectionModule]
-		})
+			imports: [MapLayoutSelectionModule],
+			declarations: [MapLayoutSelectionComponent]
+		}).compileComponents()
+		fixture = TestBed.createComponent(MapLayoutSelectionComponent)
+		loader = TestbedHarnessEnvironment.loader(fixture)
```

Comment:
do we really need to dig into Angular's internals? I will check, if I can get the tests working again with testing-library style.

Reply:
s. above